### ISSUE TITLE
Publish Diagnostics not working

### ISSUE THREAD

**Issue by ArunGanesh88** [CodinGame/languageserver-mutualized#8]
Hi, 

The implementation works well when I try to run the language server in a MAC or a Linux Machine when I try to run it in a windows Machine the syntax highlighting is not happening.  When I looked into the logs, identified we are not receiving the Publish Diagnostics details after open document.

Thanks,

**Comment by CGNonofr** [comment]
Hi,

What is running under windows? the client? the language server? both? where does the code of this project run?

**Comment by ArunGanesh88** [comment]
All 3 are in windows machine, Java language server, Monaco client and the project.

**Comment by CGNonofr** [comment]
> All 3 are in windows machine, Java language server, Monaco client and the project.

It's probably related to a file path matching (with `\` and `/`), will you have some time to investigate? I don't have any windows machine :shrug:

**Comment by ArunGanesh88** [comment]
> > All 3 are in windows machine, Java language server, Monaco client and the project.
> 
> It's probably related to a file path matching (with `\` and `/`), will you have some time to investigate? I don't have any windows machine 🤷

Yes, even I was thinking the same some thing to do with the path... 
I can investigate. Let me know the things that i should check...

**Comment by CGNonofr** [comment]
I'm unable to reproduce on windows with the example (even with `file://` uris instead of `inmemory`)

Are you?

**Comment by ArunGanesh88** [comment]
I am still Facing this issue, still it is not working in a windows Machine for me.

**Comment by CGNonofr** [comment]
I guess, there's no reason for it to suddenly work.

But until you'll be able to help me to reproduce it, I won't be able to help you
